web/helpers/authentication_helpers.ex

```diff
@@ -51,15 +56,23 @@ defmodule CodeCorps.AuthenticationHelpers do
   defp do_init_changeset(model, method, params), do: apply(model, method, params)
 
   defp get_resource(conn, model, :update) do
-    resource_name = model |> Module.split |> List.last |> Macro.underscore |> String.to_atom
+    resource_name =
+      model
+      |> Module.split
+      |> List.last
+      |> Macro.underscore
+      |> String.to_atom
+
     conn.assigns |> Map.get(resource_name)
   end
   defp get_resource(_conn, model, :create), do: model.__struct__
 
   defp get_changeset_method(action), do: "#{action}_changeset" |> String.to_atom
```

Comment:
@JoshSmith I was hoping this would establish a convention, much in the same way how canary already uses a convention to load a resource. However, we could extend the code to allow for an option to specify the method name.